### DISCUSSION TITLE
Checkpointing

### DISCUSSION THREAD

**Discussion by petrposik** [EC-KitY/EC-KitY#14] [General]
Dear authors, 

thanks for sharing your library with us! I am considering whether I should invest time to try out your library. I am wondering what features distinguishing EC-KitY from other libraries would push me towards this decision.

- Is there any speed comparison on a few typical problems comparing the time demands of EC-KitY to those of, say, DEAP?
- Is there any feature that not many other EC packages offer? For me, such a feature would be e.g. checkpointing: the ability to run for example 10^4 evaluations of certain system, examine the results so far, and if found that running the EC system longer would be desirable, load the results so far and continue with the evolution for e.g. additional 3*10^4 evaluations.

In any case, I wish you good luck in further development!

Petr

**Comment by moshesipper** [comment]
Hey Petr,
Good to hear from you.
- We believe the speed is comparable to DEAP -- our major benefits are in various other features.
- We're working on a paper -- in the meantime we've shared the table comparing **EC-KitY** with 8 other packages in the README file, so it's on the project's main page
- Checkpointing is currently on a soon-to-do list...

Cheers, 
Moshe

**Reply by petrposik** [reply]
Hi Moshe. Thanks for the response. I will try to keep an eye on this package. Looking forward to future versions! Petr
